source/documentation/10-switching-to-production.md

```diff
@@ -192,156 +225,191 @@ Account activation is completed during ePDQ account creation.
+1. On the __Users__ page, select __New User__.
+1. Complete the __UserID__ and __User’s name__ fields.
+1. Complete the __email address__ field with the email that should receive notifications about this account.
+1. Select __Admin__ in the __Profile__ field.
+1. Check the __Special user for API (no access to admin.)__ option.
 1. Enter your own password.
-1. Click _Create_.
+1. Select __Create__.
 1. You will then create a password for the API user:
-    - Enter your own password.
-    - Enter a password for the API user.
-    - Confirm the password for the API user.
-    - Click _Submit_.
-1. You will see a message that your password has been successfully updated. Click _Back to List_.
+    - enter your own password
+    - enter a password for the API user
+    - confirm the password for the API user
+    - select __Submit__
+1. You will see a message that your password has been updated. Select __Back to List__.
 
-You have now successfully set up your ePDQ account to work with GOV.UK Pay.
+>Note the API user’s username and password as you must copy them into the GOV.UK Pay account credentials page. 
 
 ### Set up 3D Secure for ePDQ
 
-To enable 3D Secure payment authentication, sign in to the [GOV.UK Pay self-service admin site](https://selfservice.payments.service.gov.uk/), click _3D Secure_ and turn 3D Secure on.
+3D Secure is an additional security layer for online credit and debit card transactions. Card payment organisations offer it to customers under a variety of names e.g. Verified by Visa. 
 
-3D Secure should be enabled by default on the [ePDQ admin site](https://payments.epdq.co.uk/Ncol/Prod/BackOffice/login/index). To check this, on the homepage, go to _Configuration > Payment methods_ and select _3D Secure column_.
+To enable 3D Secure payment authentication:
 
-## Smartpay setup
-
-To set up Smartpay to work with GOV.UK Pay, you must:
+1. Sign in to the [GOV.UK Pay self-service admin site](https://selfservice.payments.service.gov.uk/).
+1. Select __3D Secure__ and turn 3D Secure on.
 
-- set up your notification credentials
-- configure your user profile
-- configure server communication
+3D Secure should be enabled by default on the [ePDQ admin site](https://payments.epdq.co.uk/Ncol/Prod/BackOffice/login/index). To check, go to __Configuration > Payment methods__ and select __3D Secure column__.
 
-### Set up notification credentials
+### Set up live account credentials
 
 1. Go to the [GOV.UK Pay admin site](https://selfservice.payments.service.gov.uk/login).
 1. Sign in to your GOV.UK Pay account.
-1. Go to _Settings > Account Credentials > Edit notification credentials_.
-1. Enter a unique username and password (the password must contain more than 10 characters).
-1. Click _Save credentials_.
+1. Go to __My Services__ and select the live service you want to set up.
+1. Go to __Settings > Account Credentials > Edit credentials__.
+1. Complete the fields on this page:
+    - __PSP ID__ - enter your PSP ID for ePDQ
```

Comment:
Not clear on where to obtain this information. @jonathanglassman mentioned that @alexbishop1 may know

Reply:
The PSP ID is basically the name of the ePDQ account. For live accounts, I believe it’s automatically generated by ePDQ when the live account is created.

Reply:
OK, in that case I don't personally think anything else is needed here